app.js

```diff
@@ -63,19 +63,14 @@ app.get('/api/v1/palettes/:id', async (req, res) => {
 
 
 app.post('/api/v1/projects/', async (req, res) => {
-  const { name } = req.body
-  const projectAlreadyExists = await database('projects').where({name})
-
-  try {
-    if(!name) {
-      return res.status(422).json('Please name your project');
-
-    }
-    if (projectAlreadyExists.length !== 0) {
-      return res.status(422).json(`Project name with ${name} already exists. Please provide a unique name`)
-    }
-      
-    const id = await database('projects').insert({ name }, 'id')
+  const { name } = req.body;
+  const duplicated = await database('projects').where('name', name).first();
+
+  if (!name) return res.status(422).json({ error: 'Please name your project' });
+  if (duplicated) return res.status(422).json({ error: `Project name with ${name} already exists. Please provide a unique name` });
```

Comment:
I like the way you approached the unique name issue
generally I don't reach for one line `if` statements, but I understand it is consistent with our codebase.